src/Task2Test.kt

```diff
@@ -0,0 +1,42 @@
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+
+internal class Task2Test {
+
+    @Test
+    //Exceptional case where index -1 is returned
+    // to show that an array of length of less than
+    // 2 was given as input.
+    fun onEmptyArray() {
+        val (max1,max2) = intArrayOf().firstTwoLargestIndexes()
+
+        assertEquals(Int.MIN_VALUE, max1)
+        assertEquals(Int.MIN_VALUE, max2)
+    }
+
+    @Test
+    // Same bad input scenario.
```

Comment:
Not helpful. Scenario (AAA) should be reflected by test fun name. 

When to use comments: https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/